src/test/utils/seed.ts

```diff
@@ -1,7 +1,7 @@
 export default async (app: any) => {
   const [user, user2] = await Promise.all([
-    app.service('users').create({ email: 'auth@user.com' }),
-    app.service('users').create({ email: 'auth2@user.com' }),
+    app.service('users').create({ email: 'auth@user.com', name: 'John Doe' }),
+    app.service('users').create({ email: 'auth2@user.com', name: 'Jane Doe' }),
```

Comment:
`John Doe 2` to distinguish them? 🙂

Reply:
Second one is Jane, not John - but probably needs better distinction if you didn't notice :D

Reply:
Fixed